nadlogar/problems/models/zaporedja.py

```diff
@@ -0,0 +1,56 @@
+import random
+
+import sympy
+from django.db import models
+
+from .meta import Problem
+
+
+class SplosniClenZaporedja(Problem):
+    """Naloga za iskanje splošnega člena poljubnega zaporedja."""
+
+    default_instruction = "Poišči predpis za splošni člen, ki mu zadoščajo začetni členi zaporedja $@cleni$"
+    default_solution = "$a_n = @resitev$"
+
+    zamik_alternirajoce = models.BooleanField(
+        "zamaknjeno in alternirajoče zaporedje",
+        help_text="Določa, če so v izbor predpisa vljučena zamaknjena zaporedja iz kvadratov in  kubov ter alternirajoča zaporedja.",
```

Comment:
Vsi parametri nekaj določajo, tako da bi lahko na krajše, npr. "So v izbor predpisa vkjučena ..."

Reply:
Dobro. Zaenkrat sem samo tehnično prilagodil [izvirno nalogo](https://github.com/ursa16180/generiranje-nalog/blob/2a7c84f9b4842b7c63136cd2e6b8a68000394576/zaporedja.py#L110), v vsebino se niti nisem spuščal.
Naj torej po svoji presoji popravimo tudi vsebinske in slogovne pomanjkljivosti? Koliko pa so te naloge preverjene?

Reply:
Zdaj, ko imamo odprt #78, lahko načeloma ta pull request sprejmem, pa se popravlja v sklopu ostalih. Kaj točno ste mislili pod preverjanjem nalog? Super bi bilo, če imate nek konkreten spisek zahtev, ki jih je treba preveriti pri vsaki nalogi. To bi se potem lahko dopisalo v `CONTRIBUTING.md`, kot piše v #78.